dist/alpine/Dockerfile

```diff
@@ -2,16 +2,15 @@ FROM python:3.8-alpine3.12 as k8s_build
 ARG KOPS_VERSION
 ARG HELM_VERSION
 RUN \
-	mkdir /main && \
+	set -x; > /etc/inittab && mkdir /main && \
```

Comment:
`set -x` can be useful to display the actual command that is running. A bit unclear on what is the purpose for removing `/etc/inittab` though. :thinking: